src/utils/user_utils.js

```diff
@@ -108,21 +112,18 @@ export function filterProfilesMatchingTerm(users: Array<UserProfile>, term: stri
         const email = (user.email || '').toLowerCase();
         const nickname = (user.nickname || '').toLowerCase();
 
-        let emailPrefix = '';
         let emailDomain = '';
         const split = email.split('@');
-        emailPrefix = split[0];
         if (split.length > 1) {
             emailDomain = split[1];
         }
 
-        return username.startsWith(lowercasedTerm) ||
-            first.startsWith(lowercasedTerm) ||
+        return username.startsWith(trimmedTerm) ||
+            full.startsWith(trimmedTerm) ||
```

Comment:
What happened to first name?

Reply:
Since full starts with first, there was no reason to check first explicitly. For same reason, I removed the check for emailPrefix since checking email accomplished the same.